bot/constants.py

```diff
@@ -1,703 +1,728 @@
-    domain_leads: int
-    events_lead: int
-    helpers: int
-    moderators: int
-    mod_team: int
-    owners: int
-    project_leads: int
-
-    jammers: int
-
-    patreon_tier_1: int
-    patreon_tier_2: int
-    patreon_tier_3: int
-
-
-class Guild(metaclass=YAMLGetter):
-    section = "guild"
+DuckPond = _DuckPond()
 
-    id: int
-    invite: str  # Discord invite, gets embedded in chat
 
-    moderation_categories: List[int]
-    moderation_channels: List[int]
-    modlog_blacklist: List[int]
-    reminder_whitelist: List[int]
-    moderation_roles: List[int]
-    staff_roles: List[int]
+class _PythonNews(EnvConfig):
+    EnvConfig.Config.env_prefix = "python_news."
+
+    channel: int = Webhooks.python_news.channel
+    webhook: int = Webhooks.python_news.id
+    mail_lists = ['python-ideas', 'python-announce-list', 'pypi-announce', 'python-dev']
+
+
+PythonNews = _PythonNews()
+
+
+class _VoiceGate(EnvConfig):
+    EnvConfig.Config.env_prefix = "voice_gate."
+
+    bot_message_delete_delay = 10
+    minimum_activity_blocks = 3
+    minimum_days_member = 3
+    minimum_messages = 50
+    voice_ping_delete_delay = 60
+
+
+VoiceGate = _VoiceGate()
+
+
+class _Branding(EnvConfig):
+    EnvConfig.Config.env_prefix = "branding."
+
+    cycle_frequency = 3
 
 
-class Keys(metaclass=YAMLGetter):
-    section = "keys"
+Branding = _Branding()
 
-    github: Optional[str]
-    site_api: Optional[str]
 
+class _VideoPermission(EnvConfig):
+    EnvConfig.Config.env_prefix = "video_permission."
 
-class URLs(metaclass=YAMLGetter):
-    section = "urls"
+    default_permission_duration = 5
+
+
+VideoPermission = _VideoPermission()
+
+
+class _Redis(EnvConfig):
+    EnvConfig.Config.env_prefix = "redis."
+
+    host = "redis.default.svc.cluster.local"
+    password = ""
+    port = 6379
+    use_fakeredis = False  # If this is True, Bot will use fakeredis.aioredis
+
+
+Redis = _Redis()
+
+
+class _CleanMessages(EnvConfig):
+    EnvConfig.Config.env_prefix = "clean."
+
+    message_limit = 10_000
+
+
+CleanMessages = _CleanMessages()
+
+
+class _Stats(EnvConfig):
+    EnvConfig.Config.env_prefix = "stats."
+
+    presence_update_timeout = 30
+    statsd_host = "graphite.default.svc.cluster.local"
+
+
+Stats = _Stats()
+
+
+class _Cooldowns(EnvConfig):
+    EnvConfig.Config.env_prefix = "cooldowns."
+
+    tags = 60
+
+
+Cooldowns = _Cooldowns()
+
+
+class _Metabase(EnvConfig):
+    EnvConfig.Config.env_prefix = "metabase."
+
+    username = ""
+    password = ""
+    base_url = "http://metabase.default.svc.cluster.local"
+    public_url = "https://metabase.pythondiscord.com"
+    max_session_age = 20_160
+
+
+Metabase = _Metabase()
+
+
+class _BaseURLs(EnvConfig):
+    EnvConfig.Config.env_prefix = "urls."
 
     # Snekbox endpoints
-    snekbox_eval_api: str
-    snekbox_311_eval_api: str
+    snekbox_eval_api = "http://snekbox.default.svc.cluster.local/eval"
+    snekbox_311_eval_api = "http://snekbox-311.default.svc.cluster.local/eval"
```

Comment:
These are out of date, new values are:
snekbox_eval_api: `http://snekbox-310.default.svc.cluster.local/eval`
snekbox_311_eval_api: `http://snekbox.default.svc.cluster.local/eval`

Reply:
Tadaa a839fa59d9fae8151401e309eda29dc680503bed